ng-dev/release/publish/test/branch-off-next-branch-testing.ts

```diff
@@ -71,6 +72,11 @@ export async function expectBranchOffActionToRun(
   expectedNewBranch: string,
 ) {
   const action = setupReleaseActionForTesting(actionType, active, isNextPublishedToNpm);
+
+  spyOn(action.instance, '_promptForNewNextVersion' as any).and.resolveTo(
+    semverInc(active.next.version, 'preminor', 'next'),
+  );
```

Comment:
Also, if we go with that prompt, then I think we should somehow manage to get tests for the version computation, and also some tests for the major variant. Right now we only test minor, and we replicate the `preminor` next bump manually here (which is not really good in terms of coverage I think).